src/coreclr/vm/gcheaputilities.cpp

```diff
@@ -169,7 +169,7 @@ HMODULE LoadStandaloneGc(LPCWSTR libFileName)
     libPath.Append(libFileName);
 
     LPCWSTR libraryName = libPath.GetUnicode();
-    LOG((LF_GC, LL_INFO100, "Loading standalone GC from path %S\n", libraryName));
```

Comment:
This looks sketchy. If you call GetUnicode, and then GetUTF8(), I would expect the GetUnicode() pointer to become invalid.

Reply:
Yes, that is a good catch. I need to audit these changes again.

Reply:
I went through the rest of these changes. None seem to suffer from the same behavior.